kinetics_MO.py

```diff
@@ -0,0 +1,261 @@
+import numpy as np
+import pandas as pd
+import pyomo.environ as pyo
+from measure_optimize import (
+    MeasurementOptimizer,
+    SensitivityData,
+    MeasurementData,
+    CovarianceStructure,
+    ObjectiveLib,
+)
+import pickle
+import time
+
+
+# set up problem formulation
+
+# number of time points for DCM
+Nt = 8
+
+# maximum manual measurement number for each measurement
+max_manual_num = 10
+# minimal measurement interval
+min_interval_num = 10.0
+# maximum manual measurement number for all measurements
+total_max_manual_num = 10
+
+# index of columns of SCM and DCM in Q
+static_ind = [0, 1, 2]
+dynamic_ind = [3, 4, 5]
+# this index is the number of SCM + nubmer of DCM, not number of DCM timepoints
+all_ind = static_ind + dynamic_ind
+num_total_measure = len(all_ind)
+
+# meausrement names
+all_names_strategy3 = [
+    "CA.static",
+    "CB.static",
+    "CC.static",
+    "CA.dynamic",
+    "CB.dynamic",
+    "CC.dynamic",
+]
+
+# define static costs
+static_cost = [2000, 2000, 2000, 200, 200, 200]  # CA  # CB  # CC  # CA  # CB  # CC
```

Comment:
What are the units for the costs? Please explain in the comments. Also, please move the CA, CB, CC, ... comment to the line above. This will be easier to read.

Reply:
added comments